### PR TITLE
Rewrite for better handling of services that have gone down

### PR DESCRIPTION
**Pull request by woutslakhorst**
Lessons learnt from chaos testing!

### CODE REVIEW

File: impl/src/main/kotlin/nl/nuts/consent/bridge/corda/CordaManagedonnectionFactory.kt [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import nl.nuts.consent.bridge.ConsentBridgeRPCProperties
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.config.AbstractFactoryBean
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+/**
+ * abstraction for creating Corda RPC connection,
+ */
+class CordaManagedConnectionFactory : AbstractFactoryBean<CordaManagedConnection>() {
```
Comment (reinkrul) [inline impl/src/main/kotlin/nl/nuts/consent/bridge/corda/CordaManagedonnectionFactory.kt]: Would this be a candidate for a health indicator so we can monitor Corda connections?

Comment (woutslakhorst) [reply impl/src/main/kotlin/nl/nuts/consent/bridge/corda/CordaManagedonnectionFactory.kt]: There is a dedicated one in CordaFlowHealthIndicators: CordaConnectionHealthIndicator

File: impl/src/main/kotlin/nl/nuts/consent/bridge/pipelines/CordaStateMachineToNatsPipeline.kt [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+                            os.close()
+                            event.error = os.toString()
+
+                            // publish with new state
+                            val jsonBytes = Serialization.objectMapper().writeValueAsBytes(event)
+                            publish("consentRequestErrored", jsonBytes)
+                        })
+                        update.result.doOnSuccess(Consumer {
+                            // update event to suc6 status
+                            // event.name = EventName.EventConsentRequestFlowSuccess
```
Comment (reinkrul) [inline impl/src/main/kotlin/nl/nuts/consent/bridge/pipelines/CordaStateMachineToNatsPipeline.kt]: todo?

Comment (woutslakhorst) [reply impl/src/main/kotlin/nl/nuts/consent/bridge/pipelines/CordaStateMachineToNatsPipeline.kt]: That event was removed from the state diagram, it was a dead-end state. Is always overridien by another state.

**Pull request merged**